lib/data_update_scripts/20200917125741_re_index_listing_with_originally_published_at.rb

```diff
@@ -0,0 +1,7 @@
+module DataUpdateScripts
+  class ReIndexListingWithOriginallyPublishedAt
+    def run
+      Listing.find_each(&:index_to_elasticsearch_inline)
```

Comment:
We only need to reindex the published listings, as they are the ones that get updated by the other script. Maybe we should actually combine them:

```
def run
  return unless Listing.column_names.include?("originally_published_at")

  published_listings = Listing.where(published: true).select(:id)
  published_listings.in_batches(of: 100) do |batch|
       batch.update_all('originally_published_at = created_at')
       Search::BulkIndexWorker.set(queue: :default).perform_async("Listing", batch.ids)
  end
end
```

I think I like this last version the best.